bootstrap/apollo-env.sh

```diff
@@ -14,3 +17,24 @@ if [[ -z "${color_start-}" ]]; then
   declare -r color_norm="${color_start}0m"
 fi
 
+# change logging levels of called components at a global level
+# if unset allow for existing selective logging of components
+case "${APOLLO_LOG}" in
+  "")
+    # Do nothing in this instance
+  ;;
+  0)
+    # force minimal logging
+    echo "Forcing reduction of component logging"
+    unset TF_LOG
+  ;;
+  1)
+    export TF_LOG=1
+    export ANSIBLE_LOG="-v"
+  ;;
+  *)
+    export TF_LOG=${APOLLO_LOG}
+    export ANSIBLE_LOG="-vvvv"
```

Comment:
Not sure this is gonna work for vagrant as it is expecting 'vvvv' no '-vvvv' see http://docs.vagrantup.com/v2/provisioning/ansible.html